compiler/src/main/scala/org/scalajs/core/compiler/GenJSExports.scala

```diff
@@ -627,6 +618,29 @@ trait GenJSExports extends SubComponent { self: GenJSCode =>
       }
     }
 
+    private def reportCannotDisambiguateError(alts: List[Exported]): Unit = {
+      val currentClass = currentClassSym.get
+
+      // Find a position that is in the current class for decent error reporting
+      val pos = alts.collectFirst {
+        case ExportedSymbol(sym) if sym.owner == currentClass => sym.pos
+        case alt: ExportedBody                                => alt.pos
+      }.getOrElse {
+        currentClass.pos
+      }
```

Comment:
So, er ... actually I cannot come up with any source program that would enter the `getOrElse`. The reason is that `genJSClassDispatchers` is only called for `JSName`s that were found in the current class, on concrete methods. So by construction there will be at least one `alt` coming from the current class.

That said, removing the `getOrElse` means I would need a brutal `.get` or a `throw` in the `getOrElse`. It doesn't feel right to do either, when there is a perfectly reasonable thing to do in `getOrElse`. It is also future-proof and "I missed some corner case"-proof.

WDYT? Should I add a comment in there telling that (we think) there is no source code that can trigger this code?